services/QuillLMS/lib/query_examples/admin_diagnostics/diagnostic-students-view.sql

```diff
@@ -4,8 +4,8 @@
            Total Query Time:        1915 ms
            Total Slot Time:         2377 ms
            BI Engine Mode Used:     FULL_INPUT
-             BI Engine Code:          
```

Comment:
There next few files have whitespace changes. Maybe you should just check these out from develop to pull them out of the diff (I'm guessing you didn't edit these files).

Reply:
will do!